packages/pwa-buildpack/lib/WebpackTools/__tests__/PWADevServer.spec.js

```diff
@@ -1,12 +1,8 @@
 jest.mock('errorhandler');
-jest.mock('fs');
```

Comment:
Not sure why this was mocked, but was causing an error with an upstream workbox dep trying to destructure a utility from `fs.promises`. Since nothing was asserting on this spy, seemed safe to remove the mock entirely.